SorterOfPersonsByAge/src/Persons.java

```diff
@@ -0,0 +1,19 @@
+public class Persons {
+    String firstName;
+    String lastName;
+    int age;
+
+
+    public Persons(String p) {
```

Comment:
There is no JavaDoc that specifies that the constructor receives a serialized person.
Please use params for the names and age instead of one string. You can split the string before the constructor is called.